### PR TITLE
feat(tExtRef): add checkOnlyBType to subscribe options

### LINKED ISSUE #77: tExtRef: Add checkOnlyBType to subscribe
I was expecting a subscribe option to have an option for: "checkOnlyBType" and I think I missed that we didn't extend this all the way as a subscription option:

https://github.com/OpenEnergyTools/scl-lib/issues/65
https://github.com/OpenEnergyTools/scl-lib/pull/66

It seems we still need to add the option to the the subscribe options and pass through to doesFcdaMeetExtRefRestrictions so the API is exposed.

### PR DESCRIPTION
**Pull request by danyill**
Closes #77

I've just copied the preceding tests and added the `checkOnlyBType` option.

I'll be most happy to receive a review and make improvements.

### CODE REVIEW

**Review by JakobVogelsang — CHANGES_REQUESTED** [review]
LGTM appart the doc for this option. I would love it to be consistent. That does not mean that the existing one is the better coice ;)

**Comment by danyill** [comment]
> LGTM appart the doc for this option. I would love it to be consistent. That does not mean that the existing one is the better coice ;)

Sorry, I have not understood your concern - could you rephrase?

I've enhanced the type `SubscribeOptions` in line with the other options I think?

https://github.com/danyill/scl-lib/blob/6423b8bb75f8700306391a76a813a33c3428d8f7/tExtRef/subscribe.ts#L13-L20

**Comment by JakobVogelsang** [comment]
Oh, sorry I was not clear I think. The same key had different comments in the different type they are used by. I think we could make the use of the library easier by equal descriptions for equal keys.

**Comment by danyill** [comment]
I have made some updates, thanks :+1:

**Review by JakobVogelsang — APPROVED** [review]
LGTM
